final_prep/annotations.rb

```diff
@@ -4,10 +4,16 @@
 
 # Build a Bear
 
+# Create a method that defines a bear and returns data about the bear as a hash.
+# Include name, age, fur, clothes and special_power as initializing arguments
 def build_a_bear(name, age, fur, clothes, special_power)
+  # define greeting var that uses it's name
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  # define a demographics array that includes name and age
   demographics = [name, age]
+  # define a power saying phrase that includes special power
   power_saying = "Did you know that I can #{special_power}?"
+  #create a hash collecting various data about the bear
```

Comment:
There is some value in going line by line through this hash value as well- where do the values come from? Are they an argument passed into the method, or a previously declared variable?